src/chrome/content/rules/AAAS.org.xml

```diff
@@ -4,23 +4,54 @@
 		- Science_Careers.org.xml
 		- Sciencemag.org.xml
 
-
-	Nonfunctional hosts in *aaas.org:
-
-		- (www.)? ¹
-		- archives ²
-		- promo	³
-		- vivo ¹
-
-	¹ Dropped
-	² Plaintext reply
-	³ Shows default page
+	Problematic hosts in *aaas.org:
+
+		- ^ (timeout)
+		- archives (tls error)
+		- arctic (mismatch)
+		- atlas (tls error)
+		- case (tls error)
+		- hrdata (tls error)
+		- idp01 (mismatch)
+		- oascentral (mismatch)
+		- project2061 (expired)
+		- secureapps (cert chain)
+		- zserver (tls error)
+	
+	There are a further 50+ subdomains that do not support HTTPS.
 
 -->
 <ruleset name="AAAS.org (partial)">
 
+	<target host="aaas.org" />
+	<target host="www.aaas.org" />
+	<target host="account.aaas.org" />
+	<target host="annualmeeting.aaas.org" />
+	<target host="careerdevelopment.aaas.org" />
+	<target host="communityfellows.aaas.org" />
+	<target host="doserjournalismaward.aaas.org" />
+	<target host="earlycareeraward.aaas.org" />
+	<target host="entersjawards.aaas.org" />
+	<target host="fellowshipapp.aaas.org" />
+	<target host="inventionambassadors.aaas.org" />
+	<target host="leshnerfellows.aaas.org" />
+	<target host="lorealfwis.aaas.org" />
+	<target host="masonaward.aaas.org" />
+	<target host="massmedia.aaas.org" />
+	<target host="myprofile.aaas.org" />
+	<target host="oncallscientists.aaas.org" />
+	<target host="owa.aaas.org" />
+	<target host="promo.aaas.org" />
```

Comment:
I get 404 for https://promo.aaas.org while the http version redirects to https://pubs.aaas.org/Promo/promo_setup_rd.asp

Reply:
Hmm. Not sure how that slipped through. Should be fixed now. :)

Reply:
Thanks. Next time please don't rebase commits after a reviewer started reviewing, we will squash them in the end anyway.